src/lib/badges/Badge.svelte

```diff
@@ -1,26 +1,24 @@
 <script lang="ts">
   import classNames from 'classnames';
-  import CloseButton from '../utils/CloseButton.svelte';
 
   export let color: 'blue' | 'dark' | 'red' | 'green' | 'yellow' | 'indigo' | 'purple' | 'pink' = 'blue';
   export let large: boolean = false;
+  export let bordered: boolean = false;
```

Comment:
We use `border` prop on the `Frame`, so it'd be good to be consistent and use the name `border`

Reply:
I use `bordered` here is to keep it consistent with existing `large` and `rounded`, these three words are all adjectives.
@shinokada which one do you prefer?

Reply:
To be consistent with other components, it will be better to keep the `border`. If you change it, it will affect other components and pages.

Reply:
Okay, got it. 😃